crowbar_framework/app/controllers/application_controller.rb

```diff
@@ -34,6 +34,10 @@ class ApplicationController < ActionController::Base
     Rails.env.test?
   }
   before_filter :upgrade, if: proc {
+    if File.exist?("/var/lib/crowbar/upgrade/7-to-8-upgrade-compute-nodes-postponed")
+      flash[:warning] = "Some nodes are not upgraded yet. " \
```

Comment:
are flash's translatable somewhere?

Reply:
I'm not sure about flash messages, but validation ones are sometimes set for translation...
I chose not to this time, as other  validation strings in the same file are not translated.

It's a bit messy.

Reply:
Are flash message shown to the user in the UI? Then we might want to localize the message I would say.

Reply:
That's the same as for validation strings - in the end, they are presented by flash messages.

And those defined in this file (the old ones, not the one I've added) are not set for translation...